### PR TITLE
Install.rst: Add link to pep440/version-specifiers

### PR DESCRIPTION
**Pull request by thisch**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
If you look at the preview at https://github.com/thisch/documentation/blob/bfd02f96fa8caa21dd51b8b72186fc202c2b3a6c/Users/Install.rst , this doesnt render correctly.

**Comment by thisch** [comment]
Thx for making me aware of that. It should be fixed now.

**Comment by supergr35** [comment]
Needs rebase please. This should be ready after that

File: Users/Install.rst [hunk 1]
```diff
@@ -299,7 +299,8 @@ Installation Errors
 In case you are getting
 ``ValueError:('Expected version spec in', 'appdirs ~=1.4.0', 'at',
 ' ~=1.4.0')``, then don't panic. It happens when you are using an outdated
-version of pip that doesn't support our version specifiers yet.
+version of pip that doesn't support our `version specifiers
+<https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_ yet.
 
 
     Ideally, you have to create a virtual environment with a newer pip:
```
Comment (jayvdb) [inline Users/Install.rst]: Unrelated to your patch, this renders badly.  Could you check if it was intentional; if it wasnt, a newcomer issue can be created.

**Comment by jayvdb** [comment]
ack ef1949d

**Comment by jayvdb** [comment]
Needs a rebase.

File: Users/Install.rst (hunk 1 above)
Comment (thisch) [inline Users/Install.rst]: @jayvdb, where did you check this? Did you build the docs locally?

Comment (jayvdb) [inline Users/Install.rst]: I click on the "View" button here on https://github.com/coala/documentation/pull/440/files , which takes me to
 
https://github.com/thisch/documentation/blob/53b01c4f8786862ad25adb5595f60ed32f842a79/Users/Install.rst

And I look at it... ;-)

**Comment by jayvdb** [comment]
reack 3be10b7

**Comment by jayvdb** [comment]
@rultor merge

File: Users/Install.rst (hunk 1 above)
Comment (thisch) [inline Users/Install.rst]: For me, the version specifiers link renders fine.
vs

**Pull request merged**